mgr/src/server/plugins/volume_expand.cr

```diff
@@ -145,6 +145,10 @@ put "/api/v1/pools/:pool_name/volumes" do |env|
 
   set_volume_metrics(req)
 
+  # Add only the first node for fix-layout service
+  services = add_fix_layout_service(services, pool.not_nil!.name, req.name, nodes[0],
```

Comment:
@aravindavk while implementing migrate-data,
Here is it right to  add nodes[0] which is the first node of newly expanded brick.
It should/be better if it fix-layout service is to be made running in existing_nodes[0] or rollback_volume[0] (combined nodes) since that is the node which has the correct reference for fix-layout process?

Reply:
> @aravindavk while implementing migrate-data I went through this again,
> 
> Here is it right to  add nodes[0] which is the first node of newly expanded brick.
> 
> It should/be better if the fix-layout service is made to run in existing_nodes[0] or rollback_volume[0] (combined nodes) since that is the node which has the correct reference for fix-layout process?

For fix layout first existing node. And for migrate data, first node of every dist groups (new dist group is not required but you can't identify the new nodes)